compiler/rustc_parse/src/parser/stmt.rs

```diff
@@ -287,7 +292,6 @@ impl<'a> Parser<'a> {
                         stmt_span,
                         "try placing this code inside a block",
                         format!("{{ {} }}", snippet),
-                        // Speculative; has been misleading in the past (#46836).
```

Comment:
Oh wait, maybe I shouldn't have removed this. I thought it was referring to something like `Applicability::Speculative` that was removed...